src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -118,6 +128,10 @@ public function denormalize($data, $class, $format = null, array $context = [])
         try {
             return \DateTime::class === $class ? new \DateTime($data, $timezone) : new \DateTimeImmutable($data, $timezone);
         } catch (\Exception $e) {
+            if ($context[self::IGNORE_INVALID_DATETIMES] ?? false) {
```

Comment:
```suggestion
            if ($context[self::IGNORE_INVALID_DATETIMES] ?? $this->defaultContext[self::IGNORE_INVALID_DATETIMES]) {
```

You must also set `$this->defaultContext[self::IGNORE_INVALID_DATETIMES]` to `false` in the constructor.

Reply:
This change was also made on the PR :-)